### ISSUE TITLE
Adaptive captcha missing translation when resolving captcha 

### ISSUE THREAD

**Issue by btlechowski** [brave/brave-browser#18856]
Follow up to https://github.com/brave/brave-browser/issues/15600

## Steps to Reproduce

1. Clean profile
1. Run Brave in Polish PL locale
1. Enable Rewards and Ads
1. Schedule a captcha for your payment ID
1. Trigger unblinded tokens refill
1. Click `Rozwiąż`

## Actual result:

Missing translations
![image](https://user-images.githubusercontent.com/34715963/137868577-4a2e9d19-1c27-465f-834d-f87da1a81db0.png)
![image](https://user-images.githubusercontent.com/34715963/137868642-fcae3adc-6660-4b41-966c-5c20f5190eb4.png)

## Expected result:
No missing translations

## Reproduces how often: 

Easily reproduced

## Brave version (brave://version info)

Brave | 1.31.85 Chromium: 95.0.4638.50 (Official Build) (64-bit)
-- | --
Revision | 3e9db37693d0cf8e31a89fbcf50401ae91125cef-refs/branch-heads/4638@{#816}
OS | Ubuntu 18.04 LTS

cc @emerick @tmancey @jsecretan @brave/legacy_qa @rebron @mkarolin

**Comment by emerick** [comment]
@gpestana I think this would be implemented server-side, right?

**Comment by gpestana** [comment]
Correct, the locale logic is implemented in the server side since the iframe payload generated on the server. We don't have the locale logic implemented yet, although it is top priority work once we roll out adaptive captchas to production.

Related to  https://github.com/brave-intl/bat-reputation/issues/52
